src/redux/books/booksSlice.js

```diff
@@ -1,30 +1,44 @@
-import { createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+// const appId = 'Pv07lkhuAJvBV0UHCald';
+const appId = '8jyxEDElW0vb1k0o9GE1';
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
 
 const initialState = {
-  bookItems: [
-    {
-      id: 'item1',
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
-    },
-    {
-      id: 'item3',
-      title: 'The Selfish Gene',
-      author: 'Richard Dawkins',
-      category: 'Nonfiction',
-    },
-  ],
-  count: 4,
+  bookItems: [],
   isloading: true,
 };
 
+export const getBookItems = createAsyncThunk('books/getBookItems', async (thunkAPI) => {
```

Comment:
[REQUIRED] It is a requirement for you to name this action creator `fetchBooks`. Kindly update your code to use this